### ISSUE TITLE
Some of the Preferences widgets don't adapt well to smaller window sizes

### ISSUE THREAD

**Issue by algernon** [keyboardio/Chrysalis#963]
![Screenshot from 2022-06-25 13-48-20](https://user-images.githubusercontent.com/17243/175772289-aa069db7-c791-4cfd-ac84-072720af6540.png)

Screenshot says it all.

**Comment by algernon** [comment]
It's not only Preferences, but a couple of other places too. The Changelog looks silly at 800x600 too, among other things.

**Comment by obra** [comment]
mui has some layout-width dependent properties we can use for stuff like this.

**Comment by algernon** [comment]
> mui has some layout-width dependent properties we can use for stuff like this.

Yep, but for these, we didn't need to, see #969. We could still use those layout-width dependent props to improve Preferences further (reduce the right margin in the tab on smaller window sizes), but that's much lower on my prio list than fixing actual layout bugs.
